server/src/main/java/org/spine3/server/reflect/EventHandlerMethod.java

```diff
@@ -152,15 +145,19 @@ private static boolean isEventHandler(Method method) {
         }
 
         private static boolean acceptsCorrectParams(Method method) {
-            final Class<?>[] parameterTypes = method.getParameterTypes();
-
-            if (parameterTypes.length != EVENT_HANDLER_PARAM_COUNT) {
+            final Class<?>[] paramTypes = method.getParameterTypes();
+            final int paramCount = paramTypes.length;
+            if (paramCount == 1) {
+                final boolean isCorrect = Message.class.isAssignableFrom(paramTypes[MESSAGE_PARAM_INDEX]);
```

Comment:
Yes, this way we can avoid one more check. After extracting, I have to check param count one more time before.